machine_management/control_plane_machine_management/cpmso-troubleshooting.adoc

```diff
@@ -33,7 +33,7 @@ include::modules/cpmso-ts-mhc-etcd-degraded.adoc[leveloffset=+1]
 [id="cpmso-troubleshooting-shiftstack-upgrade_{context}"]
 == Upgrading clusters that run on {rh-openstack}
 
-For clusters that run on {rh-openstack-first} that you upgrade from {product-title} 4.13 to 4.14, you might have to perform post-upgrade tasks before you can use control plane machine sets.
+For clusters that run on {rh-openstack-first} that you upgrade from {product-title} 4.14 to 4.15, you might have to perform post-upgrade tasks before you can use control plane machine sets.
```

Comment:
I will check in with dev on this, but this should probably not be changed (or should be changed in a different way). It's likely that 4.13 here is the relevant bit.